torchrl/envs/libs/vmas.py

```diff
@@ -170,37 +251,132 @@ def _build_env(
 
         return env
 
-    @set_gym_backend("gym")
+    def _get_default_group_map(self, agent_names: List[str]):
+        # This function performs the default grouping in vmas
+        # Agents with names "str_int" will be grouped in group name "str"
+        # If any of the agents to not follow the naming convention, we fall back
+        # to all agents in one group named "agents"
+        group_map = {}
+        follows_convention = True
+        for agent_name in agent_names:
+            # See if the agent follows the convention "name_int"
+            agent_name_split = agent_name.split("_")
+            if len(agent_name_split) == 1:
+                follows_convention = False
+            try:
+                int(agent_name_split[-1])
+            except ValueError:
+                follows_convention = False
```

Comment:
I'd rather not use `try/except` in this case. What about `str.isdigit()`?
```suggestion
        follows_convention = agent_name_split[-1].isdigit()
```